johnhof/IceBuddy
# Timing/Period functionality for game tracker

@DuaneGarber I want to get this in now. If I try to to it all at once it will be totally unreasonable to review.

The Gist:
- styling updates
- added
    -  scss mixins
    - `<stepper>` directive used to step integer values (may be useful later)
- implemented
    - timer and period transition functionality
    - dialog/modal styling (using ngDialog https://github.com/likeastore/ngDialog)
    - team select and display


to test
- go to `/#/tracker`
- select teams
- select teams or chose `default`
- check console that teams printed match teams selected
- test everything in the tracker header
- make sure teams are displaying
- test team variations and display
